<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ra</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitterH">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="DebugWidget" name="debugTree" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QSplitter" name="splitterV">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="FieldWidget" name="field">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QPushButton" name="btnOpen">
               <property name="text">
                <string>Open</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="goToLastPosition">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="LogSlider" name="logManager" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="LogWidget" name="log" native="true"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>905</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Game">
    <property name="title">
     <string>&amp;Game</string>
    </property>
    <widget class="QMenu" name="simulatorSetup">
     <property name="title">
      <string>Simulator Setup</string>
     </property>
    </widget>
    <addaction name="actionEnableTransceiver"/>
    <addaction name="actionChargeKicker"/>
    <addaction name="separator"/>
    <addaction name="simulatorSetup"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>Logging</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="actionRecordLogLog"/>
    <addaction name="actionSave20s"/>
    <addaction name="actionBackloglog"/>
    <addaction name="actionUseLocation"/>
    <addaction name="actionChangeLocation"/>
   </widget>
   <widget class="QMenu" name="menuTesting">
    <property name="title">
     <string>Demo</string>
    </property>
    <addaction name="actionSimulator"/>
    <addaction name="actionInternalReferee"/>
    <addaction name="actionInputDevices"/>
    <addaction name="actionAutoPause"/>
   </widget>
   <widget class="QMenu" name="menuInstant_Replay">
    <property name="title">
     <string>Instant Replay</string>
    </property>
    <addaction name="actionGoLive"/>
    <addaction name="actionStepBack"/>
    <addaction name="actionStepForward"/>
    <addaction name="actionFrameBack"/>
    <addaction name="actionFrameForward"/>
    <addaction name="actionTogglePause"/>
    <addaction name="actionShowBacklog"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Logfile"/>
    <addaction name="openLogUidString"/>
    <addaction name="exportVision"/>
    <addaction name="getLogUid"/>
    <addaction name="actionLogCutter"/>
    <addaction name="separator"/>
    <addaction name="actionSaveConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPlaySpeed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Play Speed</string>
    </property>
    <addaction name="actionSpeed1"/>
    <addaction name="actionSpeed5"/>
    <addaction name="actionSpeed10"/>
    <addaction name="actionSpeed20"/>
    <addaction name="actionSpeed50"/>
    <addaction name="actionSpeed100"/>
    <addaction name="actionSpeed200"/>
    <addaction name="actionSpeed1000"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_Game"/>
   <addaction name="menuTesting"/>
   <addaction name="menuVideo"/>
   <addaction name="menuInstant_Replay"/>
   <addaction name="menuPlaySpeed"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolCommands">
   <property name="windowTitle">
    <string>Commands</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionEnableTransceiver"/>
   <addaction name="actionChargeKicker"/>
  </widget>
  <widget class="QDockWidget" name="dockReferee">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Internal RefBox</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="RefereeWidget" name="referee"/>
  </widget>
  <widget class="QDockWidget" name="dockRefereeinfo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Referee Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="RefereeInfoWidget" name="refereeinfo"/>
  </widget>
  <widget class="QDockWidget" name="dockSimulator">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Simulator</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="SimulatorWidget" name="simulator"/>
  </widget>
  <widget class="QDockWidget" name="dockInput">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Input devices</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="InputWidget" name="input"/>
  </widget>
  <widget class="QDockWidget" name="dockVisualization">
   <property name="windowTitle">
    <string>Visualizations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="VisualizationWidget" name="visualization"/>
  </widget>
  <widget class="QToolBar" name="toolTesting">
   <property name="windowTitle">
    <string>Testing</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSimulator"/>
   <addaction name="actionInternalReferee"/>
   <addaction name="actionInputDevices"/>
  </widget>
  <widget class="QToolBar" name="toolConfig">
   <property name="windowTitle">
    <string>Configuration</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConfiguration"/>
   <addaction name="actionPlotter"/>
   <addaction name="actionRecord"/>
   <addaction name="actionRecordLogLog"/>
   <addaction name="actionSaveBacklog"/>
   <addaction name="actionSaveBackloglog"/>
  </widget>
  <widget class="QDockWidget" name="dockTiming">
   <property name="windowTitle">
    <string>Timing</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="TimingWidget" name="timing"/>
  </widget>
  <widget class="QDockWidget" name="dockStrategy">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>69</width>
     <height>42</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Strategies</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="StrategySelectionWidget" name="strategies">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockRobots">
   <property name="windowTitle">
    <string>Robots</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="RobotSelectionWidget" name="robots"/>
  </widget>
  <widget class="QDockWidget" name="dockOptions">
   <property name="windowTitle">
    <string>Options</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="OptionsWidget" name="options"/>
  </widget>
  <widget class="QDockWidget" name="dockBlueDebugger">
   <property name="windowTitle">
    <string>Blue Debugger</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="DebuggerConsole" name="blueDebugger"/>
  </widget>
  <widget class="QDockWidget" name="dockYellowDebugger">
   <property name="windowTitle">
    <string>Yellow Debugger</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="DebuggerConsole" name="yellowDebugger"/>
  </widget>
  <widget class="QDockWidget" name="dockReplay">
   <property name="windowTitle">
    <string>Replay</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="ReplayTeamWidget" name="replay"/>
  </widget>
  <widget class="QDockWidget" name="dockSimConfig">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Simulator Config</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="SimulatorConfigWidget" name="simulatorConfig">
    <property name="enabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarInfo">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAboutUs"/>
   <addaction name="actionGitInfo"/>
  </widget>
  <widget class="QDockWidget" name="dockFieldParameters">
   <property name="windowTitle">
    <string>Field Parameters</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="FieldParameters" name="fieldParameters"/>
  </widget>
  <action name="actionPlotter">
   <property name="text">
    <string>&amp;Plotter</string>
   </property>
  </action>
  <action name="actionEnableTransceiver">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Enable &amp;transceiver</string>
   </property>
   <property name="toolTip">
    <string>Enable transceiver</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDisableTransceiver">
   <property name="text">
    <string>Disable transceiver</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Record file</string>
   </property>
   <property name="toolTip">
    <string>Records what you see on the field, until
toggled again (disables the backlog when active)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionChargeKicker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Charge kicker</string>
   </property>
   <property name="toolTip">
    <string>Charge kicker</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionSimulator">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Simulator</string>
   </property>
   <property name="toolTip">
    <string>Simulator</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionInternalReferee">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Internal Referee</string>
   </property>
   <property name="toolTip">
    <string>Internal Referee</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionInputDevices">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Input devices</string>
   </property>
   <property name="toolTip">
    <string>Input devices</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
  <action name="actionAutoPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pause, if unfocused</string>
   </property>
  </action>
  <action name="actionSaveBacklog">
   <property name="text">
    <string>20s</string>
   </property>
   <property name="toolTip">
    <string>Saves the last 20s of simulation
If you are currently playing a backlog in Horus mode,
you are precisely watching this 20s, so this button saves
what you are watching.</string>
   </property>
  </action>
  <action name="actionSaveBackloglog">
   <property name="text">
    <string>20s (log)</string>
   </property>
   <property name="iconText">
    <string>20s (log)</string>
   </property>
   <property name="toolTip">
    <string>Does nothing in Ra mode, and saves the last 20s
of continuous playback (including replay).</string>
   </property>
  </action>
  <action name="actionSave20s">
   <property name="text">
    <string>Save last 20s</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionBackloglog">
   <property name="text">
    <string>Save last 20s (log)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionUseLocation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Logfile default location</string>
   </property>
  </action>
  <action name="actionChangeLocation">
   <property name="text">
    <string>Select Logfile default locations</string>
   </property>
  </action>
  <action name="actionGoLive">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Go Live</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </action>
  <action name="actionStepBack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Step back</string>
   </property>
   <property name="shortcut">
    <string>Q</string>
   </property>
  </action>
  <action name="actionStepForward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Step forward</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionFrameBack">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Frame back</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionFrameForward">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Frame forward</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionShowBacklog">
   <property name="text">
    <string>Show Backlog</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionTogglePause">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Toggle Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionSpeed1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed 1%</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionSpeed5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed 5%</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionSpeed10">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed 10%</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
  <action name="actionSpeed50">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed 50%</string>
   </property>
   <property name="shortcut">
    <string>5</string>
   </property>
  </action>
  <action name="actionSpeed100">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed 100%</string>
   </property>
   <property name="shortcut">
    <string>6</string>
   </property>
  </action>
  <action name="actionSpeed200">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed 200%</string>
   </property>
   <property name="shortcut">
    <string>7</string>
   </property>
  </action>
  <action name="actionSpeed1000">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed 1000%</string>
   </property>
   <property name="shortcut">
    <string>8</string>
   </property>
  </action>
  <action name="actionSpeed20">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Speed 20%</string>
   </property>
   <property name="shortcut">
    <string>4</string>
   </property>
  </action>
  <action name="rules2017">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>2017</string>
   </property>
  </action>
  <action name="rules2018">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>2018</string>
   </property>
  </action>
  <action name="actionSave2Configuration">
   <property name="text">
    <string>Save configuration</string>
   </property>
  </action>
  <action name="actionOpen_Logfile">
   <property name="text">
    <string>Open Logfile ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionOpen_Logcutter">
   <property name="text">
    <string>Open Logcutter</string>
   </property>
  </action>
  <action name="actionLogCutter">
   <property name="text">
    <string>Log Cutter</string>
   </property>
  </action>
  <action name="actionSaveConfiguration">
   <property name="text">
    <string>Save Configuration</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionRecordLogLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Record file</string>
   </property>
   <property name="toolTip">
    <string>Records what you see on the field, until
toggled again (disables the backlog when active)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="exportVision">
   <property name="text">
    <string>Export as Vision Log</string>
   </property>
  </action>
  <action name="getLogUid">
   <property name="text">
    <string>Get LogUID</string>
   </property>
  </action>
  <action name="openLogUidString">
   <property name="text">
    <string>Open a log via UID String</string>
   </property>
  </action>
  <action name="actionAboutUs">
   <property name="text">
    <string>AboutUs</string>
   </property>
  </action>
  <action name="actionGitInfo">
   <property name="text">
    <string>GitInfo</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VisualizationWidget</class>
   <extends>QWidget</extends>
   <header>widgets/visualizationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TimingWidget</class>
   <extends>QWidget</extends>
   <header>widgets/timingwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FieldWidget</class>
   <extends>QGraphicsView</extends>
   <header>widgets/fieldwidget.h</header>
  </customwidget>
  <customwidget>
   <class>LogWidget</class>
   <extends>QWidget</extends>
   <header>widgets/logwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogSlider</class>
   <extends>QWidget</extends>
   <header>widgets/logslider.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DebugWidget</class>
   <extends>QWidget</extends>
   <header>widgets/debugwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RefereeWidget</class>
   <extends>QWidget</extends>
   <header>refereewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimulatorWidget</class>
   <extends>QWidget</extends>
   <header>simulatorwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>InputWidget</class>
   <extends>QWidget</extends>
   <header>inputwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StrategySelectionWidget</class>
   <extends>QWidget</extends>
   <header>strategyselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RobotSelectionWidget</class>
   <extends>QWidget</extends>
   <header>robotselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OptionsWidget</class>
   <extends>QWidget</extends>
   <header>optionswidget.h</header>
  </customwidget>
  <customwidget>
   <class>DebuggerConsole</class>
   <extends>QWidget</extends>
   <header>widgets/debuggerconsole.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ReplayTeamWidget</class>
   <extends>QWidget</extends>
   <header>widgets/replayteamwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RefereeInfoWidget</class>
   <extends>QWidget</extends>
   <header>refereeinfowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimulatorConfigWidget</class>
   <extends>QWidget</extends>
   <header>simulatorconfigwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FieldParameters</class>
   <extends>QWidget</extends>
   <header>fieldparameters.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSimulator</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockSimulator</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInternalReferee</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockReferee</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInputDevices</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockInput</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>802</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSimulator</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockSimConfig</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>437</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
